libraries/suggestion-input-view/src/main/java/com/trendyol/suggestioninputview/SuggestionInputEditText.kt

```diff
@@ -0,0 +1,31 @@
+package com.trendyol.suggestioninputview
+
+import android.content.Context
+import android.graphics.Canvas
+import android.graphics.Rect
+import android.util.AttributeSet
+import androidx.appcompat.widget.AppCompatEditText
+
+class SuggestionInputEditText(context: Context, attrs: AttributeSet) : AppCompatEditText(context, attrs) {
+
+    private var suffix = ""
+    private val suffixPaddingLeft = context.resources.getDimensionPixelSize(R.dimen.horizontal_padding_suggestion_item)
+    private val suffixRect = Rect()
+
+    fun setSuffix(suffix: String) {
+        this.suffix = suffix
+    }
+
+    override fun onMeasure(widthMeasureSpec: Int, heightMeasureSpec: Int) {
+        paint.getTextBounds(suffix, 0, suffix.length, suffixRect)
```

Comment:
is this necessary? it seems like we're just calculating suffixRect bounds and doing nothing about it.

Reply:
> necessary

deleted necessary codes, done